native/spark-expr/src/sum_decimal.rs

```diff
@@ -286,18 +284,16 @@ struct SumDecimalGroupsAccumulator {
     sum: Vec<i128>,
     result_type: DataType,
     precision: u8,
-    scale: i8,
 }
 
 impl SumDecimalGroupsAccumulator {
-    fn new(result_type: DataType, precision: u8, scale: i8) -> Self {
```

Comment:
`scale` was unused but the core crate suppresses dead code warnings so we did not know until this was moved